contacts.py

```diff
@@ -628,7 +631,34 @@ def scan_status(self, data, args):
         bounding_boxes = map(lambda l: (l['min_lat'], l['min_long'], l['max_lat'], l['max_long']),
                              req_locations) if req_locations else None
         number_to_return = int(self.config.get('MAX_SCAN_COUNT', 50))
-        return self._status_or_sync(prefixes, bounding_boxes, since, now, number_to_return)
+        return self._scan_or_sync(prefixes, bounding_boxes, since, now, number_to_return)
+
+    # status/result POST
+    @register_method(route='/status/result')
+    def status_result(self, data, args):
+        update_tokens = data.get('update_tokens')
+        # TODO-114 think thru health notification HEALTHY cos on client need to clear out other prior indications that infected. see note in manual contact tracing spec
+        floating_seconds = current_time()
+        self.send_or_sync({
+            "contact_ids": [{
+                "id": data.get('id'),
+                "status": data.get('status')-1,  # i.e. INFECTED=1 so send 0 to make Alice set her status to 1,
+                "update_token": update_tokens.pop(),
+                "message": data.get('message')
+            }] },
+            floating_seconds=floating_seconds
+        )
+        self._update_or_result(
+            length=len(update_tokens),
+            serial_number=2, # send_or_sync will use serial_number=0 and poss 1
```

Comment:
wonder if we want send_or_sync to return the number it did so we don't have to guess.

Reply:
Good idea - I'll change this